content/hacking-atom/sections/package-word-count.md

```diff
@@ -392,12 +434,13 @@ Next we get the number of words by calling [`getText()`](https://atom.io/docs/ap
 
 Finally, we tell our view to update the word count it displays by calling the `setCount()` method on our view and then showing the modal again. Since that method doesn't yet exist, let's create it now.
 
-We can add this code to the end of our `your-name-word-count-view.coffee` file:
+We can add this code to the end of our `your-name-word-count-view.js` file:
 
-```coffee
-setCount: (count) ->
-  displayText = "There are #{count} words."
-  @element.children[0].textContent = displayText
+```javascript
+setCount(count) {
+  const displayText = `There are ${count} words.`;
+  return this.element.children[0].textContent = displayText;
```

Comment:
I don't think the `return` is needed here.

Reply:
Thanks @jasonrudolph! just fixed these!